app/src/main/java/br/com/compass/compassmart/ui/onboarding/OnboardingFragment.kt

```diff
@@ -0,0 +1,86 @@
+package br.com.compass.compassmart.ui.onboarding
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.widget.ViewPager2
+import br.com.compass.compassmart.R
+import br.com.compass.compassmart.databinding.FragmentOnboardingBinding
+import br.com.compass.compassmart.ui.onboarding.model.InformacoesModel
+
+class OnboardingFragment : Fragment() {
+
+    private val binding: FragmentOnboardingBinding by lazy {
+        FragmentOnboardingBinding.inflate(layoutInflater)
+    }
+    private var data = mutableListOf<InformacoesModel>()
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+
+        conteudo()
+        binding.onboardingMainViewPager.adapter = OnboardingAdpter(data)
+
+            binding.onboardingMainViewPager.registerOnPageChangeCallback(object : ViewPager2.OnPageChangeCallback() {
+
+            override fun onPageScrolled(
+                position: Int,
+                positionOffset: Float,
+                positionOffsetPixels: Int
+            ) {
+                super.onPageScrolled(position, positionOffset, positionOffsetPixels)
+                mudarCorBolinhaProgresso()
+            }
+            override fun onPageSelected(position: Int) {
+                super.onPageSelected(position)
+            }
+            override fun onPageScrollStateChanged(state: Int) {
+                super.onPageScrollStateChanged(state)
+                mudarCorBolinhaProgresso()
+            }
+        })
+        return binding.root
+    }
+    // Adiciona os conteudos dos cards
+    fun adicionaNaLista(titulo : String, descricao : String, imagem: Int){
```

Comment:
Função deve ser privada, pois só é usada dentro dessa fragment. Ficar atento aos modificadores de visibilidade
```suggestion
    private fun adicionaNaLista(titulo : String, descricao : String, imagem: Int) {
```

Reply:
Alterado para função privada.